jetty/src/main/scala/org/http4s/server/jetty/JettyBuilder.scala

```diff
@@ -388,3 +393,15 @@ object JettyBuilder {
 }
 
 private final case class Mount[F[_]](f: (ServletContextHandler, Int, JettyBuilder[F]) => Unit)
+
+case class PrefixedService[F[_]](service: HttpRoutes[F], prefix: String)
+class PrefixedServices[F[_]] private (val services: List[PrefixedService[F]])
+
+object PrefixedServices {
+  def apply[F[_]](service: HttpRoutes[F], prefix: String) =
+    new PrefixedServices[F](List(PrefixedService(service, prefix)))
+
+  implicit def prefixedServicesSemigroup[F[_]]: Semigroup[PrefixedServices[F]] =
+    (x: PrefixedServices[F], y: PrefixedServices[F]) =>
+      new PrefixedServices[F](x.services ++ y.services)
+}
```

Comment:
All of this seems as germane to Tomcat as it does to Jetty.  Would it be better in the servlet package?